CHANGES.rst

```diff
@@ -5,13 +5,25 @@ Changes for Crate Admin Interface
 Unreleased
 ==========
 
-- Fixed the table/column display to no longer display all columns as being
-  generated columns.
+2019/09/16 1.15.2
+=================
+
+Changes
+-------
 
 - Replaced the Slack icon with Discourse in the help section, updated nametags
   for all languages and translations for Spanish.
 
 
+Fixes
+-----
+
+- Adapted a query for compatibility with the upcoming CrateDB 4.0 release.
```

Comment:
where is this entry coming from?

Reply:
Jordi fixed one query acording the the commit history. 

https://github.com/crate/crate-admin/commit/e305700c6071c5ed3dfdc6b3ee0046267a5089e4

Reply:
Ah, but it is already mentioned, my bad.